### PR TITLE
Build Improvement and Modifications to CPP

### PR DESCRIPTION
**Pull request by vibhatha**

### CODE REVIEW

**Comment by vibhatha** [comment]
Added a single build script to install CPP, PYARROW and PYTHON. 
Also added flags to build the project in chosen build modes (release, debug)

File: cpp/src/twisterx/python/ctx/twisterx_context_wrap.cpp [hunk 1]
```diff
@@ -0,0 +1,101 @@
+//
+// Created by vibhatha on 5/15/20.
+//
```
Comment (supunkamburugamuve) [inline cpp/src/twisterx/python/ctx/twisterx_context_wrap.cpp:3]: Lets remove created by comment

Comment (vibhatha) [reply cpp/src/twisterx/python/ctx/twisterx_context_wrap.cpp:3]: I will remove that.

Comment (vibhatha) [reply cpp/src/twisterx/python/ctx/twisterx_context_wrap.cpp]: removed

**Pull request merged**